stories/exampleTypes/index.js

```diff
@@ -256,6 +267,34 @@ export default () =>
         onBeforeRender,
       }
     )
+    .addWithJSX(
+      'Geo filter & HERE maps',
+      () => (
+        <div
+          style={{
+            backgroundColor: '#333',
+            color: '#AAA',
+            padding: '20px',
+            borderRadius: '10px',
+          }}
+        >
+          <Provider tree={testTree}>
+            <Flex style={{ flexFlow: 'column wrap' }}>
+              <div style={{ flex: 1 }}>
```

Comment:
Do we need this flex component?

Reply:
Yes we do since it helps stretch the controls full width in their parent container.